src/types.h

```diff
@@ -455,12 +466,20 @@ class SizedType {
   friend std::ostream &operator<<(std::ostream &, const SizedType &);
   friend std::ostream &operator<<(std::ostream &, Type);
 
+  SizedType &IntoPointer()
```

Comment:
I feel like this function has the potential to be a bit confusing - it's modifying the type in place, but also returning a type. On my first reading of the call site I assumed that this returned type was a newly created value. Is there a benefit to this current design?

Reply:
No particular benefit, I needed to get the `SizedType` right after converting the ref into a pointer.
I've changed it to return `void` so it doesn't look like it's returning a new type anymore.